src/android/KnoxPluginEnabled.kt

```diff
@@ -101,6 +104,20 @@ class KnoxPlugin : CordovaPlugin() {
                 }
             }
 
+            ACTION_OPEN_WIFI_SETTINGS -> {
+                cordova.threadPool.execute {
```

Comment:
Should probably follow same design as the other actions and move this logic block into a private method.
Also, the semicolon can be removed.